RecoLocalTracker/SiPixelDigiReProducers/plugins/SiPixelDigiMorphing.cc

```diff
@@ -60,7 +59,7 @@ SiPixelDigiMorphing::SiPixelDigiMorphing(edm::ParameterSet const& conf)
                                           << " number of ROCs\n";
   }
 
-  if (ncols_r_ + 2 * iters_ <= int(sizeof(uint64_t) * 8)) {
+  if (ncols_r_ + 2 * iters_ <= int(sizeof(uint64_t))) {
```

Comment:
Good catch.
And many thanks for updating the PR.

Reply:
Opps, validation is crashing on this. The "*8" was correct here; instead, the text of the error message was missing this multiplication.